modules/custom/az_block_types/az_custom_menu_block/src/EventSubscriber/AzCustomMenuBlockKernelViewSubscriber.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drupal\az_custom_menu_block\EventSubscriber;
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ViewEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Alters the block library modal.
+ *
+ * We can't use hook_ajax_render_alter() because the #markup is rendered before
+ * it is passed to that hook.
+ */
+class AzCustomMenuBlockKernelViewSubscriber implements EventSubscriberInterface {
+
+  /**
+   * The current route match.
+   *
+   * @var \Drupal\Core\Routing\RouteMatchInterface
+   */
+  protected $currentRouteMatch;
+
+  /**
+   * Constructs a new MenuBlockKernelViewSubscriber.
+   *
+   * @param \Drupal\Core\Routing\RouteMatchInterface $current_route_match
+   *   The current route match.
+   */
+  public function __construct(RouteMatchInterface $current_route_match) {
+    $this->currentRouteMatch = $current_route_match;
+  }
+
+  /**
+   * Alters the block library modal.
+   *
+   * @param \Symfony\Component\HttpKernel\Event\ViewEvent $event
+   *   The event to process.
+   */
+  public function onKernelView(ViewEvent $event) {
```

Comment:
Either this line needs to change or line 67 does.
```suggestion
  public function onView(ViewEvent $event) {
```